src/jit/optcse.cpp

```diff
@@ -2894,22 +2894,12 @@ bool Compiler::optConfigDisableCSE2()
 
 void Compiler::optOptimizeCSEs()
 {
-#ifdef DEBUG
-    if (verbose)
-    {
-        printf("\n*************** In optOptimizeCSEs()\n");
-        printf("Blocks/Trees at start of optOptimizeCSE phase\n");
-        fgDispBasicBlocks(true);
-    }
-#endif // DEBUG
-
     optCSECandidateCount = 0;
     optCSEstart          = lvaCount;
 
 #if FEATURE_VALNUM_CSE
     INDEBUG(optEnsureClearCSEInfo());
     optOptimizeValnumCSEs();
```

Comment:
I am not sure that I understand, don't you need to call 
`DoPhase(this, PHASE_OPTIMIZE_VALNUM_CSES, optOptimizeValnumCSEs);` here?

Reply:
I moved the invocation up into compiler.cpp so it was done the same way as the other optimization phases.